src/sfCommand.ts

```diff
@@ -138,10 +161,64 @@ export abstract class SfCommand<T> extends Command {
     };
   }
 
+  protected async catch(err: Error & { exitCode?: number }): Promise<Error & { exitCode?: number }> {
```

Comment:
Why name this `catch` and not just `error`?

This is totally different than the `catch` in a `try {} catch (e) {}` expression which might create unnecessary confusion.

Reply:
Its overriding catch from[ base class ](https://github.com/oclif/core/blob/main/src/command.ts#L227)